### DISCUSSION TITLE
<redacted> traces

### DISCUSSION THREAD

**Discussion by 3DTOPO** [getsentry/support#44] [Support] [answered]
So I am seeing `redacted` in back traces. If I hit the ? it says it is likely from beta SDKs.

Only, I am not using beta SDKs. I found a number of people experiencing a similar issue but was not able to find anything helpful.

Sentry isn't useful to me if I cannot see symbolicated back traces.

**Comment by 3DTOPO** [comment]
Is this the wrong place for Sentry tech support?

Or does Sentry not support their product?

**Reply by chadwhitacre** [reply]
Sorry @3DTOPO, we have processes in place for Issues but not Discussions, undergoing [some cleanup now](https://github.com/getsentry/team-ospo/issues/32). Moving this to our new Customer Support board and hopefully we get to this soon-ish. 🤞

**Comment by bruno-garcia** [comment, marked as answer]
Is that the `sentry-cocoa` SDK? What version? The SDK itself shouldn't be a reason to see `redacted`.

This can happen if we don't have the symbols on the server. It often happens on `beta` versions of iOS, that Sentry doesn't have available. Possibly simulator events too.
Or if you didn't upload the symbols for your own app. This came up before, e.g: https://github.com/getsentry/sentry-cocoa/issues/280#issuecomment-684954376
